internal/serve/auth/signature_verifier.go

```diff
@@ -40,12 +35,12 @@ type ExpiredSignatureTimestampError struct {
 }
 
 func (e ExpiredSignatureTimestampError) Error() string {
-	return fmt.Sprintf("signature timestamp has expired. sig timestamp: %s, check time %s", e.ExpiredSignatureTimestamp.Format(time.RFC3339), e.CheckTime.Format(time.RFC3339))
+	return fmt.Sprintf("timestamp expired by %s", e.CheckTime.Sub(e.ExpiredSignatureTimestamp).String())
```

Comment:
it would be helpful to add the metric after the expiry number. e.g. "timestamp expired by 5s"

Reply:
Good idea, added in https://github.com/stellar/wallet-backend/pull/154/commits/2c5a0554dbb024c4ac5e98ce1b4a7382a3971308.